src/pages/Calculator.js

```diff
@@ -337,12 +347,29 @@ export default class Calculator extends Component {
                               <td>{flavor.vendor.name}</td>
                               <td>{flavor.name}</td>
                               <td>
-                                <Button
-                                  onClick={this.addIngredient.bind(this, index)}
-                                  disabled={this.state.inuse[index]}
-                                >
-                                  <span>Add to Recipe</span>
-                                </Button>
+                                {this.state.inuse[index] && (
```

Comment:
You can use a ternary expression here for DRY purposes.

Reply:
Good idea! Fixed in https://github.com/gusta-project/frontend/pull/24/commits/75fe7f03909ec7f2601a83956091ba63bdb87608